hibernate-core/src/main/java/org/hibernate/query/results/complete/CompleteResultBuilderBasicValuedStandard.java

```diff
@@ -63,30 +63,31 @@ public CompleteResultBuilderBasicValuedStandard(
 			columnName = jdbcResultsMetadata.resolveColumnName( resultPosition + 1 );
 		}
 
+		final int jdbcPosition;
```

Comment:
I moved this into the resolver on purpose to avoid/defer accessing the JDBC metadata if possible. Is there a reason why you moved this back?

Reply:
I'm not sure what "resolver" you are talking about.  

I believe the failing case was related to `@ColumnResult( .., type=MyUserType.class )`, which we even all talked about.

Reply:
IIRC it had something to do with passing `explicitJavaTypeDescriptor` to `basicType.getExpressableJavaTypeDescriptor()`.  I'll find the test and check

Reply:
Yep, that was it.  

The test is `org.hibernate.userguide.mapping.basic.BitSetUserTypeTest#testNativeQuery`.  The code as it was did not support:

```
@ColumnResult(name = "pr.bitset", type = BitSetUserType.class)
```

This was the first fix I made was here, which resolved that problem.  But I later ran into other problems resolving native-SQL result-set mappings, which led to me also changing `ResultMementoBasicStandard`.  Those changes happened to also fix this problem.  

It seems like I can safely revert this particular change.